test/client.js

```diff
@@ -954,6 +954,8 @@ describe('CloudantClient', function() {
           auth: { username: ME, password: PASSWORD },
           method: 'GET'
         };
+
+        var responseString = '';
```

Comment:
it's a bit weird to call this responseString and then have to `toString` it later.
I think `responseData` or `responseBytes` probably makes more sense.